src/azure-cli/azure/cli/command_modules/acs/_loadbalancer.py

```diff
@@ -25,32 +25,41 @@ def set_load_balancer_sku(sku, kubernetes_version):
 
 
 def update_load_balancer_profile(cmd, managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
-                                 outbound_ports, idle_timeout, profile):
+                                 outbound_ports, idle_timeout, profile, models=None):
     """parse and update an existing load balancer profile"""
     if not is_load_balancer_profile_provided(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
                                              outbound_ports, idle_timeout):
         return profile
     return configure_load_balancer_profile(cmd, managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
-                                           outbound_ports, idle_timeout, profile)
+                                           outbound_ports, idle_timeout, profile, models=models)
 
 
 def create_load_balancer_profile(cmd, managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
-                                 outbound_ports, idle_timeout):
+                                 outbound_ports, idle_timeout, models=None):
     """parse and build load balancer profile"""
     if not is_load_balancer_profile_provided(managed_outbound_ip_count, outbound_ips, outbound_ip_prefixes,
                                              outbound_ports, idle_timeout):
         return None
 
-    ManagedClusterLoadBalancerProfile = cmd.get_models('ManagedClusterLoadBalancerProfile',
-                                                       resource_type=ResourceType.MGMT_CONTAINERSERVICE,
-                                                       operation_group='managed_clusters')
+    ManagedClusterLoadBalancerProfile = None
+    if models:
```

Comment:
remove this if/else.

I think it would be better to pass the models out of this function, and use it.
this backward support not necessary.

Reply:
Sure, changed in commit [9c4c33f](https://github.com/Azure/azure-cli/pull/19458/commits/9c4c33fa9981b3a63e3b75b29f4c61db12cd62d0) and [fd9042d](https://github.com/Azure/azure-cli/pull/19458/commits/fd9042d84e7d72554d5f6a611d708fa2f742626a).